core/src/main/java/com/airbnb/aerosolve/core/transforms/ListTransform.java

```diff
@@ -34,4 +35,13 @@ public void doTransform(FeatureVector featureVector) {
       transform.doTransform(featureVector);
     }
   }
+
+  @Override
+  public void doTransform(Stream<FeatureVector> featureVectors) {
```

Comment:
Oh yes. I did this originally for performance but then found out that `ListTransform` has to collect all the feature vectors so it can operate on them repeatedly, thus defeating the purpose. I'm going to change it to `Iterable` instead.